core/src/ast/methods.rs

```diff
@@ -30,6 +30,12 @@ pub struct Method {
 
     /// The lifetimes introduced in this method and surrounding impl block.
     pub lifetime_env: LifetimeEnv,
+
+    /// The list of `cfg` attributes (if any).
+    /// 
+    /// These are strings instead of `syn::Attribute` or `proc_macro2::TokenStream`
+    /// because those types are not `PartialEq`, `Hash`, `Serialize`, etc.
+    pub cfg: Vec<String>,
```

Comment:
thought: for the other attributes I was going to store them as meta items in the AST, but i guess this is fine for now, and easier to convert to JSON/etc

Reply:
when I make my changes I'll be moving this to an `attributes` struct